client/src/components/topics/topic-tabs.tsx

```diff
@@ -230,6 +239,31 @@ function TopicTabs(props: {
               />
             ) : null
           )}
+          {topicQuestionsLoadStatus === LoadStatus.NONE ? (
```

Comment:
[Optional] Possible refactor to a loop that returns the same component instead of having the same one three times:
```jsx
<Tab
        label="Loading topics..."
        disabled
        className="loading-topic-tab topic-tab"
/>
```